code-samples/creating-your-bot/command-handling/commands/kick.js

```diff
@@ -1,12 +1,12 @@
-const { SlashCommandBuilder } = require('@discordjs/builders');
+const { SlashCommandBuilder } = require('discord.js');
 
 module.exports = {
 	data: new SlashCommandBuilder()
 		.setName('kick')
 		.setDescription('Select a member and kick them (but not really).')
 		.addUserOption(option => option.setName('target').setDescription('The member to kick')),
 	async execute(interaction) {
-		const user = interaction.options.getUser('target');
-		return interaction.reply({ content: `You wanted to kick: ${user.username}`, ephemeral: true });
+		const member = interaction.options.getMember('target');
+		return interaction.reply({ content: `You wanted to kick: ${member.user.username}`, ephemeral: true });
```

Comment:
what's the need to change this :thinking: is it just for people in case they run user.kick?

Reply:
yeah